docs/source/architecture/sub_stakes.rst

```diff
@@ -0,0 +1,278 @@
+
+A sub-stake remains active until it becomes unlocked, and a staker gets the reward for the last period by calling ``mint()`` or ``confirmActivity()`` once the last period is surpassed. Each staker can have no more than 30 active sub-stakes which are stored in an array. All sub-stake changes initially reuse slots of inactive sub-stakes for storage in the array, and if there are none, will instead use empty slots. Therefore, attempting to retrieve data about previous inactive sub-stakes is not guaranteed to be successful since the data could have been overwritten.
+
+
+
+Operations that modify the sub-stake array
+------------------------------------------
+
+Deposit and locking
+^^^^^^^^^^^^^^^^^^^
+*Methods* : ``deposit(uint256,uint16)``,  ``deposit(address,uint256,uint16)``,  ``lock(uint256,uint16)``
+
+To become a staker, NU tokens must be transferred to the ``StakingEscrow`` contract and locked using one of the ``deposit()`` methods. If the staker already has unlocked tokens within the account in the contract, then the stake can be locked using the ``lock()`` method. If successful, a new element will be created in the array of sub-stakes with the next period as the starting date, and the duration equal to the input parameter.
+
+
+**Example:**
+
+    A staker deposits 900 tokens:
+        * 1st sub-stake = 900 tokens starting from the next period and locked for 5 periods
+
+    .. aafig::
+        :proportional:
+        :textual:
+
+            stake
+            ^
+            |
+         900|   +-------------------+
+            |   |        1st        |   period
+            +---+---+---+---+---+---+---->
+            | 0 | 1 | 2 | 3 | 4 | 5 | 6
+
+
+
+Lock prolongation
+^^^^^^^^^^^^^^^^^
+*Methods* : ``prolongStake(uint256,uint16)``
+
+In order to increase the staking reward, as well as the possibility of obtaining policies with a longer timeframe, stakers can increase the duration of their locked sub-stake using the ``prolongStake()`` method. The number of sub-stakes does not change, but the locked duration for the specified sub-stake will be increased.
+
+**Example:**
+
+    A staker prolongs sub-stake for 2 additional periods up to 7:
+		- Before: 
+			* 1st sub-stake = 900 tokens for 5 periods
```

Comment:
we should use our 'locking duration' term - similar comment for other spots in the page.

```suggestion
			* 1st sub-stake = 900 tokens with locking duration of 5 periods
```

Reply:
✔️